src/components/landingpage/cards/Cards.jsx

```diff
@@ -0,0 +1,13 @@
+function Cards({ icon, title, text }) {
```

Comment:
Implementing redux will make passing props easier. You can use my reservation slice files as an example.